spec/requests/moderation/legislation_proposals_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
```

Comment:
Aah, I did not go for the right kind of test here I guess ...
I have written a feature test instead, but I would like a clarification:
In the proposals specs, there is a test that checks that 'Hide' and 'Block author' are not visible when connected as a moderator/admin.
Here, I still can see the 'Hide' link when connected as an admin. (Which mean that I could hide my own legislation proposal).
Is this the expected behaviour ? Should I hide this link when connected as an admin/moderator ? (and test it here).

Also, there is no 'Legislation proposals' section in the moderation screen and cannot add tests like it is done for the proposals section.

Reply:
For consistency, if moderators can't hide their own proposals/debates/comments, they shouldn't be able to hide their own legislation proposals.

Regarding administratos, currently they can't hide their own proposals either, but that's something we'll change when we implement issue #1793 :thinking:.

So, to be clear:

* Moderators should not be able to hide their own legislation proposals, and there should be a test checking this scenario
* For administrators, do whatever is easier for you :wink:

Reply:
> Also, there is no 'Legislation proposals' section in the moderation screen and cannot add tests like it is done for the proposals section.

Feel free to add a new file, like `spec/features/moderation/legislation/proposals_spec.rb` :wink:.

Reply:
Hello @javierm , I have a problem here with the link. 
There is a can[:manage] defined and it takes over any other configuration.
Which means that the link hide is still there. I could have removed it by removing the can[;manage], but I fear the side effects and may be we are going outside the scope of this PR.
Furthermore, you are talking about re defining roles and profiles, so I have chosen not to play with those settings.

Reply:
If you're talking about administrators, it's fine to leave the link visible and let them hide their own proposals :ok_hand:.

Please correct me if you weren't talking about administrators :sweat_smile:; if that's the case, could you tell me which file you meant?

Reply:
Hello @javierm , we are talking about the same thing: administrators. So I guess there is nothing to change as this is the expected behaviour then.
But since the adminstrator behaviour (app/model/abilities/administrator.rb) inherits from the moderation.rb, I might have been a bit confused.